plugins/hydra_ray_launcher/tools/setup_ami.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import subprocess
+from datetime import datetime
+
+dependencies = [
+    "boto3==1.15.6",
+    "hydra-core>=1.0.0",
+    "ray==1.0.0",
+    "cloudpickle==1.4.1",
```

Comment:
Maybe it's best to depend on `ray >= 1.0.0` to catch future regressions before our users?

Reply:
sounds good!